CMakeLists.txt

```diff
@@ -127,6 +131,8 @@ IF(CONFIG_LINUX_PACKAGING_SUPPORT)
     
     IF(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
         set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "arm64")
+    ELSEIF(CMAKE_SYSTEM_PROCESSOR STREQUAL "ppc64le")
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "ppc64el")
```

Comment:
Shouldn't this be "ppc64le" (two last letters swapped) ?

Reply:
According to this, it's "el": https://wiki.debian.org/SupportedArchitectures